workday/13.0/modules/ROOT/pages/workday-connector-upgrade-migrate.adoc

```diff
@@ -8,245 +8,38 @@ Upgrade Anypoint Connector for Workday (Workday Connector) to version 13.x.x.
 [%header,cols="50a,50a"]
 |===
 |From Version | To Version
-|11.1.x |12.x.x
+|12.1 |13.x
```

Comment:
13.x or 13.x.x?
(Consider searching for "13." and verifying whether you really want 0, x, or x.x.)